content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md

```diff
@@ -98,10 +98,10 @@ your cluster. Those fields are:
 
   {{< note >}}
   The `MinDomainsInPodTopologySpread` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-  enables `minDomains` for pod topology spread. Starting from v1.28,
+  enables `minDomains` for pod topology spread. From v1.28 to v1.29,
   the `MinDomainsInPodTopologySpread` gate 
-  is enabled by default. In older Kubernetes clusters it might be explicitly
-  disabled or the field might not be available.
+  is enabled by default. 
+  And, at v1.30, this feature graduated to stable and the feature gate and the gate doesn't support the disablement anymore.
   {{< /note >}}
```

Comment:
I would drop the note, now that the feature has graduated. Someone who is looking at the wrong docs version isn't much likely to be misled as the default for all supported versions is to have the API server recognize the field, and for other components to honor it.

```suggestion
```

Reply:
What you could do instead: update the _main_ branch to have text that is appropriate for v1.29, and also hold this PR until the change from _main_ has merged in to _dev-1.30_. Then, get this change merged.

Reply:
I just removed the note as suggested